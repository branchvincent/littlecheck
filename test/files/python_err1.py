# RUN: /usr/bin/python3 %s

import sys

sys.stderr.write("ALPHA\n")
# CHECKERR: BETA

# Some superfluous output so we can test --after.
sys.stderr.write("GAMMA1\n")
sys.stderr.write("GAMMA2\n")
sys.stderr.write("GAMMA3\n")
sys.stderr.write("GAMMA4\n")
sys.stderr.write("GAMMA5\n")
sys.stderr.write("GAMMA6\n")
sys.stderr.write("GAMMA7\n")
sys.stderr.write("GAMMA8\n")
