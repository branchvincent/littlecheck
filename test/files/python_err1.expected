Failure in python_err1.py:

  The CHECKERR on line 6 wants:
    BETA

  which failed to match line stderr:1:
    ALPHA

  Context:
    ALPHA <= does not match CHECKERR on line 6: BETA
    GAMMA1 <= no more checks
    GAMMA2
    GAMMA3
    GAMMA4
    GAMMA5
    GAMMA6
    GAMMA7
    GAMMA8

  when running command:
    /usr/bin/python python_err1.py
